plugins/tools/secret_msg/__main__.py

```diff
@@ -93,7 +94,7 @@ async def inline_answer(_, inline_query: InlineQuery):
 
 
 @userge.bot.on_cmd("secretmsg", about={
-    'header': "-_-",
+    'header': "send a media in bot personal message, and reply `{tr}secretmsg`",
```

Comment:
markdown doesn't work in help section, use html

Reply:
btw where is target user who can see secret message ?

Reply:
it is below, handled in the dict